pulumi/python/README.md

```diff
@@ -1,13 +1,14 @@
 # MARA: Pulumi / Python
 
-This directory contains the
-
-This project illustrates the end to end stand up of an AWS VPC cluster, Elastic Kubernetes Service (EKS), NGINX
-Kubernetes Ingress Controller (KIC), and a sample application using [Pulumi](https://www.pulumi.com/). The project is
-intended to be used as a reference when building your own infrastructure as code deployments. As such, each discrete
-stage of deployment is defined as a separate Pulumi project that can be deployment independently of each stage. Although
-Pulumi supports many programming languages, Python was chosen as the language for this project. The reimplementation of
-the deployments definitions here should be reproducible in other languages.
+This project illustrates the end-to-end stand up of an AWS VPC cluster, Elastic 
+Kubernetes Service (EKS), NGINX Kubernetes Ingress Controller (KIC), and a sample 
+application using [Pulumi](https://www.pulumi.com/). It is intended to be used 
+as a reference when building your own Infrastructure as Code (IaC) deployments. 
+As such, each discrete stage of deployment is defined as a separate Pulumi project 
+that can be deployed independently of each stage. Although Pulumi supports many 
+programming languages, Python was chosen as the language for this project. 
+The reimplementation of the deployments definitions here should be reproducible 
```

Comment:
Is "deployments definitions" the correct phrasing? Grammatically it seems a bit off but maybe this is a common phrase.

Reply:
Go ahead and change this to "...reimplementation of the deployment here should be easily reproducible..."